lib/signs/bus.ex

```diff
@@ -318,6 +329,17 @@ defmodule Signs.Bus do
          ))
   end
 
+  defp should_read(current_time, state) do
```

Comment:
Minor, but by Elixir naming conventions we should add a `?` to the end of this function to indicate that it returns a boolean